datastore/longhorn.go

```diff
@@ -2430,6 +2429,7 @@ func (s *DataStore) IsNodeDownOrDeleted(name string) (bool, error) {
 	if cond.Status == longhorn.ConditionStatusFalse &&
 		(cond.Reason == string(longhorn.NodeConditionReasonKubernetesNodeGone) ||
 			cond.Reason == string(longhorn.NodeConditionReasonKubernetesNodeNotReady)) {
+		logrus.Debugf("Node is marked down due to %s", cond.Reason)
```

Comment:
Should not log here but the caller instead, because this is just a query method.